src/components/TodoItem.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { Todo } from '../types/Todo';
+import classNames from 'classnames';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = props => {
+  const { todo } = props;
+
+  return (
+    <div
+      data-cy="Todo"
+      className={classNames('todo', { completed: todo.completed })}
+    >
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={todo.completed}
+        />
+      </label>
+
+      <span data-cy="TodoTitle" className="todo__title">
+        {todo.title}
+      </span>
+      <button type="button" className="todo__remove" data-cy="TodoDelete">
+        ×
+      </button>
```

Comment:
The delete button should have an `onClick` handler to remove the todo item. Consider implementing this functionality to allow users to delete todos.